objects/obj_player/Step_0.gml

```diff
@@ -177,7 +177,56 @@ if (keyboard_check_pressed(ord("E"))) {
 		}
 	}
 }
+//if (bouncing == true && animation_complete) {
```

Comment:
Is there a reason for keeping this commented-out code? if so can we add a comment as to why we are keeping it? can we remove all the extra spacing below the commented-out code to leave just one line of padding?

Reply:
oh the the test code, i forgot to delete
Fixed